FV3/gfsphysics/GFS_layer/GFS_physics_driver.F90

```diff
@@ -4566,8 +4566,27 @@ subroutine GFS_physics_driver                                 &
             call set_state("total_precipitation", rain1)
             call set_state("ratio_of_snowfall_to_rainfall", Diag%sr)
             call set_state("tendency_of_rain_water_mixing_ratio_due_to_microphysics", rainp)
-            call call_function("emulation.monitor", "store_zarr")
-            call call_function("emulation.monitor", "store_netcdf")
```

Comment:
why is this removed? did this save data and you ran simulation separately in another program?

Reply:
it's been replaced by a single call to "store". Downstream users can define their own implementations of ".store".

Reply:
It's still in there down below, just simplified to `store` instead of specific routines for zarr and netCDF.